python/sdist/amici/de_export.py

```diff
@@ -941,6 +970,26 @@ def import_from_sbml_importer(
             whether to compute conservation laws
         """
 
+        # add splines as expressions to the model
+        # saved for later substituting into the fluxes
+        spline_subs = {}
+
+        for ispl, spl in enumerate(si.splines):
+            # HACK remove real=True assumptions from spline symbols
+            #   must be done as the first thing because it modifies the SbmlImporter
+            old_sbml_id = spl.sbml_id
+            new_sbml_id = sp.Symbol(spl.sbml_id.name)
+            si._replace_in_all_expressions(old_sbml_id, new_sbml_id)
```

Comment:
Why is that an issue? Wouldn't that be handled better by adding `real=True` elsewhere?

Reply:
I have no idea, this was something that was put in by Paul